packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -283,3 +281,11 @@
 		max-height: $grid-unit-60 * 2; // Adjust to not cover the save button, showing three items.
 	}
 }
+
+.dataviews-action-modal__duplicate-template-part {
```

Comment:
I didn't see where this class is added. I'm guessing it's generated. I wonder if we should rely on a generated class. I feel the "width" should be a prop of the modal and maybe a config in the action but that's for later.

Reply:
Yeah, it is generated and for now I wanted to avoid extra modal props in action. We can revisit soon with new actions needed.

Reply:
Personally I don't think it's bad to have a generated class name based on the unique action's id. I'm also not too concerned about indicating a public API as we can have breaking changes in Data Views.

Having said that, the alternative would be to have `modalProps` in actions API and that would also indicate public API. I think we'll know more when we are preparing for extending the APIs and update as needed then.